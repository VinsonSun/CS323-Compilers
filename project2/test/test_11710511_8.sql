struct Node {
   int v;
}

int test8() {
   struct Node node;
   return node;
}
