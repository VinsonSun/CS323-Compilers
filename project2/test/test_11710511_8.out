Error type 8 at Line 8: incompatiable return type
